tests/main.c

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdio.h>
+
+#include "define_enum_2_test.h"
+#include "mu_eat_empty_args.h"
+
+int main(void)
+{
+    int result;
+    result = run_define_enum_2_test();
+    if (result != 0)
+    {
+        printf("failures in run_define_enum_2_test\n");
+        return result;
+    }
+
+    result = run_mu_eat_empty_args_test();
+    if (result != 0)
+    {
+        printf("failures in run_mu_eat_empty_args_test\n");
```

Comment:
>printf [](start = 8, length = 6)

(void)

Reply:
above and everyhere.

---
In reply to: [270706215](https://github.com/Azure/azure-macro-utils-c/pull/11#discussion_r270706215) [](ancestors = 270706215)